package.json

```diff
@@ -2,7 +2,7 @@
   "name": "rotorflight-configurator",
   "productName": "Rotorflight Configurator",
   "description": "Crossplatform configuration tool for Rotorflight flight control system.",
-  "version": "0.0.0",
```

Comment:
Please don't change this. It is automatically set to the right version in the build process. Otherwise it should be 0.0.0.

Reply:
OK, I will restore this value to the default.